### PR TITLE
v2v: Pre-check install_drivers checkbox for windows VMs

### PR DESCRIPTION
**Pull request by matobet**
Depends on fix to `Dialog#init_fields_with_values` (PR https://github.com/ManageIQ/manageiq/pull/15810)

Bug-Url: https://bugzilla.redhat.com/show_bug.cgi?id=1472843

### CODE REVIEW

**Comment by matobet** [comment]
@miq-bot add_label fine/yes

**Comment by matobet** [comment]
@miq-bot assign @gmcculloug

**Comment by gmcculloug** [comment]
@matobet PR https://github.com/ManageIQ/manageiq/pull/15810 is merged, if you unwip we can merge this.

**Comment by matobet** [comment]
@miq-bot remove_label wip

File: content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/install_drivers.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        module Transform
+          module Import
+            class InstallDrivers
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                os = @handle.root['vm'].operating_system
+                is_windows = os && os.product_name =~ /windows/i
```
Comment (gmcculloug) [inline content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/install_drivers.rb]: @matobet Minor, but you could use `try` here: `os.try(:product_name) =~ /windows/i`

**Comment by matobet** [comment]
@gmcculloug done

**Comment by jelkosz** [comment]
@simaishi could you please backport this?

**Comment by simaishi** [comment]
Fine backport details:

```
$ git log -1
commit 2e49226616778e04f5a2524c3ff9807494436016
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Thu Aug 31 17:04:19 2017 -0400

    Merge pull request #170 from matobet/dynamic-windows
    
    v2v: Pre-check install_drivers checkbox for windows VMs
    (cherry picked from commit efd5d06b76ca7c581586a122ccb184c891e49b43)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1512955
```

**Pull request merged**